k8s/app_v2.Makefile

```diff
@@ -13,8 +13,21 @@ include $(makefile_dir)/var.Makefile
 
 VERIFY_WAIT_TIMEOUT = 600
 
+
+##### Common variables #####
```

Comment:
I think it makes sense to move this just before `##### Helper functions #####` section, since we have now validation after variables are used.

For example:

```
APP_GCS_PATH ?= $(GCS_URL)/$(APP_ID)/$(TRACK)

ifndef APP_ID
$(error APP_ID must be defined)
endif
```

Reply:
Good point

Reply:
I moved APP_ID check before variables definitions, but the rest must remain, because they are verified in "Validations and Information" section